src/components/Nav.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {useState} from "react";
+import { useState } from "react";
```

Comment:
Plutôt faire ça sur une ligne

```suggestion
import React, { useState } from 'react';
```